04_ocp_repo_sync.sh

```diff
@@ -37,9 +37,14 @@ sync_go_repo_and_patch github.com/openshift/machine-config-operator https://gith
 
 sync_go_repo_and_patch \
   github.com/openshift/installer \
-  https://github.com/openshift/installer.git \
-  https://github.com/tomassedovic/installer/commit/dns-workaround.patch \
-  https://github.com/flaper87/installer/commit/service-vm.patch
+  https://github.com/openshift/installer.git
+
+# NOTE(shadower): we can't just just download and apply the patch here -- the branch has several commits
+pushd "$GOPATH/src/github.com/openshift/installer"
```

Comment:
The other option is to apply the 3 patches. We can get a `patch` file per commit. It'd be pretty much like it was before but all coming from the same branch. what do you think?

Reply:
Yea I've mentioned that in the commit message. That would mean any change to this branch would require changing ocp-doit to update the commits.

I'm not keen, but we can do that instead, sure.

A third option would be to squash the service-vm commits in your branch. But only you can do that I think.

Reply:
Let's squash, lemme do that

Reply:
squashed

Reply:
Nice one, thanks!